cypress.json

```diff
@@ -1,4 +1,31 @@
 {
   "baseUrl": "http://localhost:3000",
-  "projectId": "84mdct"
+  "projectId": "84mdct",
+  "env": {
+    "state": {
+      "authenticated": true,
+      "currentUser": {
+        "user": {
+          "uid": "nXr8n9AY1RfE2bSFxjIO6a4XGCg2",
+          "displayName": "ESA Tester",
+          "email": "esa.tester@andela.com",
+          "apiKey": "AIzaSyBWJDqR1_xe7DGaOa6IwO2WxzVjJ5AXQ4A",
```

Comment:
hey @ssewilliam , why do you have apiKeys pushed to the repo.?

Reply:
Hey @bozicschucky 
thanks for raising the concerns, let me try and address this.

Reply:
Hey @bozicschucky, 

Removing the whole value of state to environment is not easy.
For now I have replaced the concerned apikey from this file, although I am not sure the value in the environment **CYPRESS_APIKEY** is is replacing the apikey.